services/director-v2/tests/unit/with_dbs/test_api_route_dynamic_services.py

```diff
@@ -232,22 +235,33 @@ def get_stack_status(node_uuid: NodeID) -> RunningDynamicServiceDetails:
         f"{module_base}._task.DynamicSidecarsScheduler.get_stack_status",
         side_effect=get_stack_status,
     )
-
     # MOCKING remove_service
-    def remove_service(node_uuid: NodeID, can_save: Optional[bool]) -> None:
+    def remove_service(
+        node_uuid: NodeID,
+        can_save: Optional[bool],
+        skip_observation_recreation: bool = False,
```

Comment:
this is a test. is it necessary to have an optional argument? does it not need to be tested as well?